sips/sip-60.md

```diff
@@ -54,24 +55,68 @@ TBD: Should the escrowed SNX be liquidated at a larger discount? or vested and t
 ```
 interface ISynthetixEscrow {
     // Views
-    
+
+    // Check whether an address has migrated from the Old reward escrow to the new contract
+    function accountMigrated(address account) external view returns (bool);
+
+    // Check whether an address has migrated their escrowed SNX to L2 staking.
+    function accountMigratedToL2(address account) external view returns (bool);
```

Comment:
I'm not sure about this. This contract will have no way of checking it - it will only be a flag that's be switched. Do we want it?

Reply:
Do we allow a user / address which has migrated to L2, to continue staking / claiming rewards on L1 if they acquire some SNX again and have rewards again on L1?